pack.py

```diff
@@ -11,8 +11,8 @@
 
 def init():
     print("===============================================")
-    print("             try to install gulp-cli globally")
-    os.system("npm install --global gulp-cli") # nosec
+    print("             try to install gulp-cli")
+    os.system("npm install gulp-cli") # nosec
```

Comment:
So, we don't need to have gulp-cli installed globally?

Reply:
yes, I think it doesn't need to do that, otherwise it will have access deny issue in the CI machine.
To change this I see the CI is good:
https://github.com/Autodesk-AutoCAD/AutoLispExt/runs/1687520696